setup.cfg

```diff
@@ -5,6 +5,22 @@ versionfile_source = shapely/_version.py
 versionfile_build = shapely/_version.py
 tag_prefix =
 
+[flake8]
+max-line-length = 88
+ignore =
+    # space before : (needed for how black formats slicing)
+    E203,
+    # too many leading '#' for block comment
+    E266,
+    # TODO line length too long
```

Comment:
Why TODO?

Reply:
Because I think we should, long term, actually remove this ignore rule and fix the lines that are too long. Currently that are mostly lines in docstrings (which black doesn't reformat), but also for docstrings it's useful to keep the line length limited.